### PR TITLE
Add com.github.carlos157oliveira.Calculus

### PR DESCRIPTION
**Pull request by carlos157oliveira**
Release for submitting to flathub.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.github.carlos157oliveira.Calculus

**Comment by LeandroStanger** [comment]
bot, build com.github.carlos157oliveira.Calculus

**Comment by carlos157oliveira** [comment]
bot, build com.github.carlos157oliveira.Calculus

**Comment by carlos157oliveira** [comment]
Using flatpak-pip-generator with requirements file produces an error when compiling. Trying to find an alternative to not use network during build time.

**Comment by barthalion** [comment]
There are other apps using matplotlib so it can't be impossible: https://github.com/search?q=org%3Aflathub+matplotlib&type=code

**Comment by carlos157oliveira** [comment]
bot, build com.github.carlos157oliveira.Calculus

File: python3-requirements.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/36/64/be592e84c443a70ea5dcfb1c30bfe6f10ba7d8eb05c2264c7ceca8498548/cppy-1.1.0.tar.gz",
+                    "sha256": "4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642"
+                }
+            ]
+        },
+        {
```
Comment (carlos157oliveira) [inline python3-requirements.json:34]:
```suggestion
        hfhfg
```

**Comment by barthalion** [comment]
bot, build com.github.carlos157oliveira.Calculus

**Comment by barthalion** [comment]
```
Processing application com.github.carlos157oliveira.Calculus
com.github.carlos157oliveira.Calculus: AppData problem: tag-invalid : <url> does not start with 'http://' [carlos157oliveira@gmail.com]
Error loading AppData file: AppData file /app/share/appdata/com.github.carlos157oliveira.Calculus.appdata.xml was not valid
```

You can validate it locally with org.freedesktop.appstream-glib from Flathub. (`flatpak run org.freedesktop.appstream-glib validate filename.appdata.xml`)

**Comment by carlos157oliveira** [comment]
bot, build com.github.carlos157oliveira.Calculus

**Comment by carlos157oliveira** [comment]
What has been done with my application? I tested the flatpak from the link provided by the bot and I found that internationalization is working (currently only for english and portuguese). However, I didn't succeed applying i18n in my build tests. Why is it working all of sudden?

**Comment by barthalion** [comment]
If you used `flatpak-builder --install` locally, there should be no difference.

**Pull request closed without merging**